manifest.toml

```diff
@@ -0,0 +1,81 @@
+packaging_format = 2
+
+id = "paperless-ngx"
+name = "Paperless-ngx"
+description.en = "Scan, index and archive all your physical documents"
+description.fr = "Scannez, triez et archivez tous vos documents papiers"
+
+version = "1.14.5~ynh2"
+
+maintainers = ["Tagada"]
+
+[upstream]
+license = "GPL-3.0-or-later"
+website = "https://paperless-ngx.com"
+demo = "https://demo.paperless-ngx.com/"
+admindoc = "https://paperless-ngx.readthedocs.io/en/latest/index.html"
+userdoc = "https://paperless-ngx.readthedocs.io/en/latest/usage_overview.html"
+code = "https://github.com/paperless-ngx/paperless-ngx"
+cpe = "???" # XXX: No CPE yet... check https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=paperless
+
+[integration]
+yunohost = ">= 11.0.0"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "350M"
+ram.runtime = "350M"
+
+[install]
+    [install.domain]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "domain"
+    full_domain = true
+
+    [install.admin]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "user"
+
+    [install.admin_pw]
+    ask.en = "Please chose a password for admin user."
+    type = "password"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "all_users"
+
+    [install.is_public_api]
+    ask.en = "Should Paperless-ngx API be accessible for apps."
+    type = "boolean"
+    default = false
+
+[resources]
+    [resources.sources]
+
+        [resources.sources.main]
+        url = "https://github.com/paperless-ngx/paperless-ngx/releases/download/v1.14.5/paperless-ngx-v1.14.5.tar.xz"
+        sha256 = "355a5760cbc967c2bd6dcaf89f577aefd5acedd908361377976e04e17c95a8f9"
+
+        autoupdate.strategy = "latest_github_release"
+        autoupdate.asset = "paperless-ngx-*.tar.xz"
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.data_dir]
+    subdirs = ["consume", "data", "media"]
+
+    [resources.permissions]
+    main.url = "/"
```

Comment:
Login is only possible, when not loggin to yunohost.

```
    main.auth_header = false
    
    api.url = "/api"
    api.show_tile = false
    api.auth_header = false
    api.allowed = "all_users"
```

Is it possible to add "visitors" to api.allowed when "install.is_public_api" is answered with true in the manifest.toml?
Or do we have to add it in the install script?

Reply:
Now we have the `init_{perm}_permission` question o/
See `allowed` here : https://yunohost.org/fr/packaging_apps_resources#properties-for-each-perm-